src/compiler/parser.ts

```diff
@@ -4926,7 +4937,7 @@ namespace ts {
 
             const fullStart = getNodePos();
             const decorators = parseDecorators();
-            const modifiers = parseModifiers();
+            const modifiers = parseModifiers([SyntaxKind.ConstKeyword]);

```

Comment:
Alternatively, if you plan to parse out more than just `const`, you can declare this once as a constant and reuse it.